frontend/src/scenes/pipeline/BatchExportBackfillModal.tsx

```diff
@@ -54,18 +56,46 @@ export function BatchExportBackfillModal({ id }: BatchExportRunsLogicProps): JSX
                 enableFormOnSubmit
                 className="space-y-2"
             >
+                {
+                    // We will assume any dates selected are in the project's timezone and NOT in the user's local time.
+                    // So, if a user of a daily export selects "2024-08-14" they mean "2024-08-14 00:00:00 in their
+                    // project's timezone".
+                }
                 <LemonField name="start_at" label="Start Date" className="flex-1">
                     {({ value, onChange }) => (
-                        <LemonCalendarSelectInput value={value} onChange={onChange} placeholder="Select start date" />
+                        <LemonCalendarSelectInput
+                            value={value}
+                            onChange={(date) => {
+                                const projectDate = date.tz(timezone, true)
+                                onChange(projectDate)
```

Comment:
@benjackwhite By any chance, do you have any opinion on this? I was quite surprised to learn we were not already assuming dates inputted here were in the project's timezone. I've made the change only in our modal, but I'm wondering if this is a problem everywhere, or if I am the one confused.

Reply:
I've almost never used that component tbh. I would assume that it is rarely used for things as specific as batch exports is using it

Reply:
For instance with most analytics queries the timezone is applied server side based on the team (which makes sense as it sends a non-timezone aware date usually.